scripts/test/run-integration-tests.sh

```diff
@@ -42,7 +42,7 @@ function run_integration_tests(){
   TEST_RESULT=success
   (cd $INTEGRATION_TEST_DIR/perpodsg && CGO_ENABLED=0 ginkgo --skip=LOCAL $EXTRA_GINKGO_FLAGS -v -timeout=35m -- -cluster-kubeconfig=$KUBE_CONFIG_PATH -cluster-name=$CLUSTER_NAME --aws-region=$REGION --aws-vpc-id $VPC_ID) || TEST_RESULT=fail
   if [[ -z "${SKIP_WINDOWS_TEST}" ]]; then
-    (cd $INTEGRATION_TEST_DIR/windows && CGO_ENABLED=0 ginkgo --skip=LOCAL $EXTRA_GINKGO_FLAGS -v -timeout=120m -- -cluster-kubeconfig=$KUBE_CONFIG_PATH -cluster-name=$CLUSTER_NAME --aws-region=$REGION --aws-vpc-id $VPC_ID) || TEST_RESULT=fail
+    (cd $INTEGRATION_TEST_DIR/windows && CGO_ENABLED=0 ginkgo --skip=LOCAL $EXTRA_GINKGO_FLAGS -v -timeout=180m -- -cluster-kubeconfig=$KUBE_CONFIG_PATH -cluster-name=$CLUSTER_NAME --aws-region=$REGION --aws-vpc-id $VPC_ID) || TEST_RESULT=fail
```

Comment:
We probably need to understand why we have to keep adding time to this test...

Reply:
I also hate to increase the testing time. In this case I am increasing the time because I added new integration tests which add about 20 minutes give or take or testing time.

What would you suggest as an appropriate timeout? If left as is there is a risk of the tests failing with a timeout error since the integration test suite has been increased with the new tests I added.

Reply:
If it is a 20 min increase in test time, let's limit it to 150m total which also gives 10m buffer. Would that be more appropriate?

Reply:
I think 150m is a good compromise. Setting it to 150m.